### ISSUE TITLE
Title is not read in screen reader

### ISSUE THREAD

**Issue by egli** [sbsdev/catalog#20]
The title of a book (which happens to be a link to the online library) is not read in the screen reader when using the standard settings in Acrobat Reader.

**Comment by egli** [comment]
I guess we could just set the alt text to the title of the book. However I'm not sure if that is really the point of the alt text. Might be best to ask Biancetti what to to do here.

**Comment by egli** [comment]
Sent a mail to Biancetti asking for advice what should be in the alt text

**Comment by egli** [comment]
The answer is not very conclusive:

> Meine Frage ist nun: Was muss im Alt-Text des Links stehen? Soll ich da wirklich den Text des Links (also den Titel des Buches) wiederholen oder ist das ein Problem von Acrobat Reader? Was ist die Empfehlung eines Accessibility-Experten?
> 
> > Antwort: Nichts!
> > 
> > Der "Alt-Text" im Link (statt korrekt und aufwändiger mit Link Description) ist ein Trick um bei der Prüfung mit PAC 2.0 ein Nullfehler barrierefreies PDF zu erhalten.
> > 
> > Bei der PAC 3.0 Version wird das nicht mehr möglich sein!

So we'll put the title in the alt text

**Comment by egli** [comment]
Fixed in 35dffed01a446f5d7796bdab073ce1dd8fdfc0a2
